web/src/main/resources/application.properties

```diff
@@ -347,9 +348,9 @@ kundportalen.access.token=accessToken
 kundportalen.subscription.adaptation.start.date=2021-09-28
 kundportalen.require.subscription.start.date=2022-05-10
 kundportalen.subscriptions.url=${kundportalen.base.url}/api/service/v1/services
-kundportalen.subscriptions.service=Intygstj�nster
+kundportalen.subscriptions.service=Intygstj?nster
```

Comment:
The text encoding seems to been changed in some way. The Swedish character isn't displayed correct. Please take a look at this as it is important that the characters are handled correct.

Reply:
Same on line 353.